main.js

```diff
@@ -1 +1,91 @@
+/* Objectives of this assignment:
 
+ -  understand how to use JS functions
+*/
+function functionName () {
+    console.log("");
+} 
+functionName
+
+// understand how to use JS algorithms
+
+for (function functionName "");
+ 
+ /* add:
+    condition
+    parameter
+    argument
+
+    Note: a statement goes here
+*/
+
+
+//  functions have to be called or invoked
+functionName ("" , "");
+
+// function output can be displayed to user and internnally to developer
+console.log ("");
+
+// or
+return ("")
+
+// * practice working with the DOM - the Document Object Model
+// * practice responding to user input
+
+
+// Code for a rectangle.
+/*  In small group on 9/16/20, we thought we needed to code a border around the calculator because that is how the .jpg appeared when viewed in VSCode but when you open the .jpg in Windows Viewer you see there is no border around the calculator */
+
+
+
+
+// Make the rectangle green inside
+
+
+
+// Put a grid inside the rectangle for the keys and output
+
+
+
+// Make the grid clickable
+
+
+// Somehow/somewhere there is a loop
+// while loop -- increments to increase to a certain number
+let i = 0;
```

Comment:
What is incrementing? Where are these numbers coming from and where are they going?